style.css

```diff
@@ -104,6 +106,220 @@
   list-style: none;
 }
 
+/* work-section */
+
+.work-section {
+  background-color: #f7f7f9;
+  width: 100%;
+  padding: 30px;
+  padding-top: 100px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
```

Comment:
- [ ] As suggested by the previous code reviewer, kindly ensure that the works section is displayed grid. The individual card contents should be displayed flex. Therefore, I also suggest that the container of all the cards is displayed grid to meet the project requirements ☺️.